src/components/StepperField/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Stepper } from '@scaleway/ui'
+import React, { ComponentProps, FocusEvent, FocusEventHandler } from 'react'
+import { useField } from 'react-final-form'
+import pickValidators from '../../helpers/pickValidators'
+import useValidation from '../../hooks/useValidation'
+import { BaseFieldProps } from '../../types'
+
+type StepperValue = NonNullable<ComponentProps<typeof Stepper>['value']>
+
+type StepperValueFieldProps<T = StepperValue, K = string> = BaseFieldProps<
+  T,
+  K
+> &
+  Partial<
+    Pick<
+      ComponentProps<typeof Stepper>,
+      | 'disabled'
+      | 'maxValue'
+      | 'minValue'
+      | 'onMaxCrossed'
+      | 'onMinCrossed'
+      | 'size'
+      | 'step'
+      | 'text'
+      | 'value'
+      | 'onChange'
+    >
+  > & {
+    name: string
+    required?: boolean
+    onBlur?: FocusEventHandler<HTMLInputElement>
+    onFocus?: FocusEventHandler<HTMLInputElement>
+  }
+
+const StepperField = ({
+  disabled,
+  maxValue,
+  minValue,
+  name,
+  onChange,
+  onBlur,
+  onFocus,
+  onMaxCrossed,
+  onMinCrossed,
+  required,
+  size,
+  step,
+  text,
+  validate,
+  value,
+}: StepperValueFieldProps) => {
+  const validateFn = useValidation<StepperValue>({
+    validate,
+    validators: pickValidators({
+      required,
+    }),
+  })
+
+  const { input } = useField(name, {
+    type: 'number',
+    validate: validateFn,
+    value,
+  })
+
+  return (
+    <Stepper
+      name={name}
```

Comment:
Missing value in props no ?

Reply:
Nop https://github.com/scaleway/scaleway-ui/blob/main/src/components/Stepper/index.tsx#L159